assets/queries/k8s/resource_with_allow_privilege_escalation/image_pull_policy_of_container_to_always/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "set_image_pull_policy_of_the_container_to_always",
+  "queryName": "Set Image Pull Policy Of The Container To Always",
```

Comment:
This is only a comment, this is not required, but I would rephrase the query name to identify the failure, instead of the expected value, and specify the expected value in keyExpectedValue